lib/editors.js

```diff
@@ -165,7 +165,7 @@ class Editor {
           type: 'reset_from_git',
           description: 'Reset state to remote git repository',
           command: 'git',
-          arguments: ['reset', '--hard', 'origin/master'],
+          arguments: ['reset', '--hard', `origin/${this.course.branch}`],
```

Comment:
I think I would have blindly written `locals` here as well, so I'm glad you got it right!

Reply:
The benefits of local pattern-matching 😛